test/integration/targets/meraki_admin/tasks/main.yml

```
@@ -57,7 +57,8 @@
       orgAccess: none
       tags:
         - { "tag": "production", "access": "read-only" }
```

Change this one as well? 
```suggestion
        - tag: production
           access: read-only
```

---

For integration tests we do not care what formatting people use.
If this was the case, we would have a lot of work on our hands.

---

You're right. I just saw the two similar blocks, one changed and one unchanged, and went into copy-editor mode.